WordPress/WordPressApi/WordPressOrgXMLRPCApi.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import wpxmlrpc
 
-public class WordPressOrgXMLRPCApi: NSObject
+public class WordPressOrgXMLRPCApi: NSObject, WordPressOrgXMLRPC
 {
     public typealias SuccessResponseBlock = (AnyObject, NSHTTPURLResponse?) -> ()
     public typealias FailureReponseBlock = (error: NSError, httpResponse: NSHTTPURLResponse?) -> ()
```

Comment:
Interesting. I'm not getting the crash, but I get different compilation errors when removing those.

I was thinking though that I'm not a big fan of typealiasing block signatures, so maybe we could remove those? The aliases were more interesting in Obj-C since block syntax was verbose and complex, but it's more straightforward in Swift.